core/crawl/crawler_thread.py

```diff
@@ -199,3 +204,20 @@ def _load_probe_json(jsn):
             print "-- JSON DECODE ERROR %s" % jsn
         except Exception:
             raise
+
+    @staticmethod
+    def _is_duplicate(probe, request, threshold=6):
```

Comment:
I would have place used the `threshold` in `constants.py`.

Reply:
I want the threshold to be modifiable if we get to use cases where it depends on the length... for all the tests that I have done until now 6 seems best suited, but it is dependent on the wivet cases.